compatibility/README.md

```diff
@@ -13,3 +13,76 @@ to be there for this workspace to use.
 Any Bazel command can then be used afterwards. `./test.sh` are the
 tests run on CI.
 
+### What is currently being tested?
+
+We include the following SDK versions in our tests:
+
+1. All stable SDK versions.
+2. All snapshots `x.y.z-snapshot-*` for which there is no stable release `x.y.z`
+3. HEAD.
+
+Since running all tests can be rather slow, we run them in a daily
+cron job. On each PR we only include HEAD and the latest stable
+release.
+
+#### Cross-version compatibility between ledger-api-test-tool
+
+We test that the `ledger-api-test-tool` of a given version passes
+against Sandbox next of another version. We test all possible version
+combinations here to ensure forwards and backwards compatibility. The
+`ledger-api-test-tool` includes a DAR built using a compiler from
+the same SDK version so this also ensures that we can load a DAR from
+a different SDK version.
+
+Since all our JVM ledger clients use the same client libraries we
+consider the `ledger-api-test-tool` to be a good proxy and if things
+are not covered it should be extended.
+
+#### Data-continuity for Sandbox classic
```

Comment:
Given the current description, it's unclear to me whether this involves creating any contract in a past version and accessing said contracts after migration. The DAR could be pure library.

Reply:
Good point, I’ve spelled out the “interactions with the ledger” a bit more explicitly. Does that help?

Reply:
👍